engines/rancher_on_eks/app/models/rancher_on_eks/deployment.rb

```diff
@@ -306,5 +296,24 @@ def deploy
       # in case Rancher create command gets failed status
       raise StandardError.new("Creating #{ApplicationController.helpers.friendly_type(@type)}: status failed") if Rails.configuration.lasso_error.present? && Rails.configuration.lasso_error != "error-cleanup"
     end
+
+    def select_zones(
+        instance_type: @cluster_size.instance_type,
```

Comment:
bit nitpicking, should that have 2 spaces indentation ?

Reply:
It's intentionally 4, so the closing parenthesis can be indented to the interior of the method. It's a weird case style-wise but I prefer this to the alternative.